Jamulus.pro

```diff
@@ -125,19 +125,19 @@ win32 {
             message(Using ASIO.)
             message(Please review the ASIO SDK licence.)
 
-            !exists(windows/ASIOSDK2) {
-                error("Error: ASIOSDK2 must be placed in Jamulus windows folder.")
+            !exists(libs/ASIOSDK2/common) {
+                error("Error: ASIOSDK2 must be placed in Jamulus \\libs folder such that e.g. \\libs\ASIOSDK2\common exists.")
```

Comment:
Ah, you added two more unescaped backslashes.

I double checked it, and it seems that qmake treats `\\x` and `\x` the same (with exception of `x = "`), so I guess you could just leave them unescaped, but it still seems cleaner to have it escaped, IMO.

Reply:
Ok.